codesign/inputparse.go

```diff
@@ -92,3 +105,52 @@ func validateCertificates(certURLList string, certPassphraseList string) ([]stri
 func splitAndClean(list string, sep string, omitEmpty bool) (items []string) {
 	return sliceutil.CleanWhitespace(strings.Split(list, sep), omitEmpty)
 }
+
+func parseFallbackProvisioningProfiles(profilesList string) ([]string, error) {
```

Comment:
so this returns a clean, valid list of profile file paths, based on a profile list (that contains file profile file paths and dirs storing profiles).

I propose updating the function name and adding some description, something like:

```
// validateAndExpandProfilePaths validates and expands profilesList.
// For directory list items, the contained profiles' path will be returned.
validateAndExpandProfilePaths
```

Reply:
Updated.